### ISSUE TITLE
My homebrew is blank! :S

### ISSUE THREAD

**Issue by PalaNolho** [stolksdorf/homebrewery#421]
Hi,

I opened one of my homebews and suddenly it was blank! Is there a way to recover it?
This is the share url: http://homebrewery.naturalcrit.com/share/HkbJ3zzS1-

Many thanks in advance
- Miguel

**Comment by calculuschild** [comment]
Closing for now, see #654.
 
Unfortunately, it looks like your brew has been deleted (shows "can't get that"). Was it just "empty" before, or did it show the "can't get that" message? Any other information you can provide would be helpful, @PalaNolho .

**Comment by PalaNolho** [comment]
@calculuschild 
in my case, initially, the brew was blank the.
I noticed that, for some reason, there was another user that was associated with my brew, it don't know how.

later, the same brew had content completely different created by that user.

Since so much time has passed since I reported, I'm not sure if there is still a backup somewhere of it, possibly not.

I deleted it from "my account" so that I I didn't mess up the brew of the other user.
It would be very happy if it would be still possible to recover it... In any case, I think that some security should be added in terms of sharing. I don't think "multiple people" editing the same file is something that will happen, but if it does, I think it should require some sort of "sharing/authorize" option so that people can't mess it up if someone get the edit link by mistake (just a suggestion)
